tests/integration/targets/rds_cluster_create/tasks/main.yaml

```diff
@@ -118,10 +118,49 @@
         that:
           - not _result_create_db_cluster.changed
 
+    # aurora-iopt1 test
```

Comment:
Verified manually that it Sets config to 
```
Cluster storage configuration
Aurora Standard
```

The issue with the tasks for `amazon.aws.rds_cluster` and `amazon.aws.rds_cluster_info` is, they do not return `storage_type` when `storage_type` param is not set or even when  set `storage_type=aurora`.

Also tried with aws cli command, however does not return StorageType as well https://docs.aws.amazon.com/cli/latest/reference/rds/describe-db-clusters.html

Reply:
If you handle the defaults in the module logic, instead of leaving them to the API, you should be able to access the returned parameters (like here https://github.com/ansible-collections/amazon.aws/pull/2063/files#diff-41d80755edc72340b8c75ea1d8c215e8e9b974de3ffe70e2bd0538c37baa2f0aR1329).

Reply:
tried handling as suggested, still unable to get the value of `stroage_type` in return values.
probably I'm missing something?

Reply:
Okay, I understand. I also tried with different versions of botocore. We should probably add a note in the RETURN block for the storage type. Does this behavior occur with the default storage type value on aurora engines or also on mysql/postgres?

@tremble Any suggestions?

Reply:
@alinabuzachis @tremble considering that api does not return the required value/param, and the I have manually verified the results, should we revert this and continue to progress the pr?